packages/manager/modules/pci/src/projects/project/streams/add/add.controller.js

```diff
@@ -0,0 +1,74 @@
+import get from 'lodash/get';
+
+import { MESSAGE_CONTAINER_NAME } from './add.constants';
+
+export default class PciStreamsAddController {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+
+    // other attributes
+    this.model = {
+      region: null,
+      kind: null,
+      name: null,
+      description: '',
+    };
+
+    this.loading = {
+      add: false,
+    };
+
+    this.messageHandler = null;
+  }
+
+  /* ================================
+  =            Callbacks            =
+  ================================= */
+
+  onStepperFinish() {
+    this.loading.add = true;
+
+    return this.addStream({
+      region: this.model.region.region,
+      kind: this.model.kind,
+      name: this.model.name,
+      description: this.model.description,
+    }).catch((error) => {
```

Comment:
No success message ?

Reply:
Seems handled with the call of `goToStreams` which allow to display a `CucCloudMessage` https://github.com/ovh-ux/manager/pull/1395/files/55d9de6a9d10ce8e38bbc8e0cd834d290ae2c4b6#diff-3132cb478fc321834f340526da25e3fcR26-R28

The success message is displayed in streams list, the error message in stream creation form (to avoid to the customer to fill it again).